tests/data/domains/movies.yaml

```diff
@@ -1,6 +1,11 @@
+name: test_movies
 slot_names:
   TITLE:
   GENRE:
   ACTOR:
   KEYWORD:
   DIRECTOR:
+
+requestable_slots:
+  - plot
+  - rating
```

Comment:
Slot names above are writen ALLCAPS. We should be consistent.
Also, shouldn't the requestable slots be a subset of all slots?

Perhaps consider for later using a special _ID field for simply requesting an item.